.importlinter

```diff
@@ -0,0 +1,67 @@
+[importlinter]
```

Comment:
Can you make the script output a comment on the first line like this:

```
# this file has been generated by scripts/generate-import-linter-config.py, do not edit by hand!
```

This should help prevent wasting time for others :)

Reply:
Very good idea! I will do it.